src/main/java/ch/njol/skript/effects/EffHidePlayerFromServerList.java

```diff
@@ -0,0 +1,90 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.effects;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Iterator;
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.ScriptLoader;
+import ch.njol.util.Kleenean;
+import com.destroystokyo.paper.event.server.PaperServerListPingEvent;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.bukkit.event.server.ServerListPingEvent;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Hide Player from Server List")
+@Description({"Hides a player from the <a href='expressions.html#ExprHoverList'>hover list</a> " +
+		"and decreases the <a href='expressions.html#ExprOnlinePlayersCount'>online players count</a> (only if the player count wasn't changed before)."})
+@Examples({"on server list ping:",
+		"	hide {vanished::*} from the server list"})
+@Since("INSERT VERSION")
+public class EffHidePlayerFromServerList extends Effect {
+
+	static {
+		Skript.registerEffect(EffHidePlayerFromServerList.class,
+				"hide [info[rmation] (of|about|related to)] %players% (in|on|from) [the] server list",
```

Comment:
the `[info[rmation] (of|about|related to)]` part is just filler that won't ever actually be used

Reply:
why not

Reply:
i don't think it's something anyone will ever type naturally

Reply:
But It makes sense :[ what about a second syntax?